### PR TITLE
add testcase check connect.EndpointConfig not nil

### PR DESCRIPTION
**Pull request by chchliang**
Signed-off-by: chchliang <chen.chuanliang@zte.com.cn>

**- What I did**
In client/network_connect_test.go  file `func TestNetworkConnect(t *testing.T) ` add branch check connect.EndpointConfig not nil

**- How I did it**

image

**- How to verify it**

### CODE REVIEW

File: client/network_connect_test.go [hunk 1]
```diff
@@ -87,6 +87,10 @@ func TestNetworkConnect(t *testing.T) {
 				return nil, fmt.Errorf("expected 'container_id', got %s", connect.Container)
 			}
 
+			if connect.EndpointConfig == nil {
```
Comment (AkihiroSuda) [inline client/network_connect_test.go]: This checking is virtually covered in `if connect.EndpointConfig.NetworkID != "NetworkID"`

Comment (chchliang) [inline client/network_connect_test.go]: I think `if  connect.EndpointConfig==nil`  then throw exception,never  reach 
`if connect.EndpointConfig.NetworkID != "NetworkID"`，so i add this branch check can be full.

Comment (vdemeester) [inline client/network_connect_test.go]: Hum, the test will fail both way (either an error, or a panic I guess). I guess it's ok to add this check, the failure would be more explicit 👼

**Review by vdemeester — APPROVED** [review]
LGTM 🐸

**Review by cpuguy83 — APPROVED** [review]
LGTM

**Pull request merged**